source/javascripts/_custom_events.js

```diff
@@ -0,0 +1,13 @@
+(function(window) {
+  function onLoad(svgFinder, callback) {
```

Comment:
svgFinder looks extremely arbitrary as the argument name here. Isn't this suposed to be a more generic thing?